scripts/quests/windurst/A_Pose_by_Any_Other_Name.lua

```diff
@@ -59,6 +60,9 @@ quest.sections =
             ['Angelica'] =
             {
                 onTrigger = function(player, npc)
+                    if xi.events.starlightCelebration.isStarlightEnabled() ~= 0 then
```

Comment:
Doesn't this block quest progression for 2 weeks? Can we check if you're actively smilebringing?

Reply:
Aye, caught this one last night as well--will update!

Reply:
Updated!